web/repos.jspf

```diff
@@ -125,12 +125,17 @@ Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
                 <%
                 if (pHelper.getRepositories(group).size() > 0 ) {
                 %>
-                    <table>
+                    <table >
+                    <col/>
```

Comment:
Avoid using `<col>` as the [attributes are mostly obsolet in html5](http://www.w3schools.com/TAgs/tag_col.asp)

Extract this into the css files (`default/style.css`, `polished/style.css`, `offwhite/style.css`). Do not forget to change all of them. You can set a `class` on the column or select it with some other more complicated selector.